backend/coins/btc/handlers/handlers.go

```diff
@@ -91,41 +89,11 @@ type FormattedAmount struct {
 	Conversions map[string]string `json:"conversions"`
 }
 
-func formatAsCurrency(amount float64) string {
-	formatted := strconv.FormatFloat(amount, 'f', 2, 64)
-	position := strings.Index(formatted, ".") - 3
-	for position > 0 {
-		formatted = formatted[:position] + "'" + formatted[position:]
-		position -= 3
-	}
-	return formatted
-}
-
-// Conversions handles fiat conversions
-func Conversions(amount coin.Amount, coin coin.Coin, isFee bool) map[string]string {
```

Comment:
The Conversions function is used twice, once by the btc handlers and once by the backend handlers. Before, the backend handlers could just call the Conversion function in the btc handlers, because it used the rates global instance. So I can either do it like I do it in this patch, or define the Conversion function twice, once as part of the backend handlers and once for the btc handlers.

Reply:
Or: expose `backend.RatesUpdater()` and have `Conversions()` take the rates updater as an additional param.

`Conversions()` used to be local to account handlers, but with the account summary the backend handlers started making use of it. At that time, moving this function to its own file would have been good. I'd moveit into `backend/coins/coin/conversions.go`, as all params are from that package, add the rates updater as a param, and use it like this from everywhere.

Reply:
Done, thanks for the description.